guides/v2.2/install-gde/basics/basics_software.md

```diff
@@ -18,7 +18,7 @@ functional_areas:
 
 The full list of software is listed in [System Requirements]({{ page.baseurl }}/install-gde/system-requirements.html) but here are the essentials:
 
-* Web server for providing access to web pages - Currently, we've documented [Apache](http://en.wikipedia.org/wiki/Apache_HTTP_Server){:target="_blank"} and plan to document other web servers in the near future.
+* Web server for providing access to web pages - Currently, we've documented [Apache](http://en.wikipedia.org/wiki/Apache_HTTP_Server){:target="_blank"} and [NGINX](https://de.wikipedia.org/wiki/Nginx){:target="_blank"}. We also plan to document other web servers in the near future.
```

Comment:
I believe that we can remove the following part. Since Apache and Nginx are documented I don't think that there will be another web server in the near future. Thank you!
```suggestion
* Web server for providing access to web pages - Currently, we've documented [Apache](http://en.wikipedia.org/wiki/Apache_HTTP_Server){:target="_blank"} and [NGINX](https://de.wikipedia.org/wiki/Nginx){:target="_blank"}.
```